examples/time_series/power_spectra_example.py

```diff
@@ -26,7 +26,7 @@
 
 x_ray = ts.columns[0]
 # The suitable value for fs would be 0.25 Hz as the time step is 4 s.
-freq, spectra = signal.periodogram(ts.data[x_ray], fs=0.25)
+freq, spectra = signal.periodogram(ts.quantity(x_ray), fs=0.25)
```

Comment:
Is this just how timeseries works?

Reply:
Yes, to get a 'column' it's recommended to use `.quantity` which returns an astropy `Quantity`

Reply:
Fancy.